daemon/config/config.go

```diff
@@ -97,6 +98,12 @@ type Config struct {
 
 	// Pidfile keeps daemon pid
 	Pidfile string `json:"pidfile,omitempty"`
+
+	// Default log configuration
+	DefaultLogConfig types.HostConfigAO0LogConfig `json:"default-log-config, omitempty"`
+
+	// RegistryService
+	RegistryService types.RegistryServiceConfig `json:"registry-service, omitempty" `
```

Comment:
We add this field in the code, however I am afraid we have not add this field's assignment in manager's code, right? Do we have a plan to add this field assignment? @ZouRui89